### PR TITLE
Simplify the Vivado install guide by pointing to Xilinx's documentation.

### PR DESCRIPTION
**Pull request by jrvanwhy**
This PR also provides a new feature selection screenshot based on the 2020.2 installer, and points to instructions for adding Vivado's tools to your shell environment.

Signed-off-by: Johnathan Van Why <jrvanwhy@google.com>

### CODE REVIEW

**Comment by jrvanwhy** [comment]
This is an alternative to #15957 

[GitHub-rendered view of the Markdown](https://github.com/jrvanwhy/opentitan/blob/vivado-install-simplify/doc/getting_started/install_vivado/index.md)

I left in the part about udev config, because I didn't see it in the Vivado install guide. That said, I'm not sure those instructions are really Vivado-specific -- wouldn't they apply just as much to ottool?

File: doc/getting_started/install_vivado/index.md [hunk 1]
```diff
@@ -2,101 +2,26 @@
...
-    Click "Next" to continue.
-
-    ![Vivado installation step 7](img/step7.png)
-
-11. Double-check the installation summary and click on "Install" to start the installation process.
+_**Vivado Version:** The recommendation is to use Vivado {{< tool_version "vivado" >}}._
 
-    ![Vivado installation step 8](img/step8.png)
+See Chapter 4 ("Download and Installation") of the [Vivado Design Suite User
+Guide](https://www.xilinx.com/content/dam/xilinx/support/documents/sw_manuals/xilinx2020_2/ug973-vivado-release-notes-install-license.pdf#namedDest=InstallingTheVivadoDesignSuiteTools)
```
Comment (a-will) [inline doc/getting_started/install_vivado/index.md:13]: https://docs.xilinx.com/r/2020.2-English/ug973-vivado-release-notes-install-license/Installing-the-Vivado-Design-Suite-Tools will get you to the section you wanted. That link doesn't work for me in Chrome's PDF viewer.

Though... maybe you meant to start from the section you named instead? https://docs.xilinx.com/r/2020.2-English/ug973-vivado-release-notes-install-license/Download-and-Installation

**Review by a-will — APPROVED** [review]
Thanks for cleaning this up! (You'll probably want someone else to approve the general direction, heh.)

File: doc/getting_started/install_vivado/index.md (hunk 1 above)
Comment (jrvanwhy) [reply doc/getting_started/install_vivado/index.md]: I did mean to start from the "Download and Installation" section. It contains the "Installing the Vivado Design Suite Tools" section, and includes instructions for downloading the installer.

Thanks for the better links, I don't know how to navigate their documentation and only found the PDF version.

**Comment by jrvanwhy** [comment]
> Thanks for cleaning this up! (You'll probably want someone else to approve the general direction, heh.)

Any suggestions on who I should add?

**Comment by jrvanwhy** [comment]
@moidx #15957 and #15985 (this PR) provide two different options for updating the Vivado installation guide to 2020.2. Which approach do you prefer?

**Review by moidx — APPROVED** [review]

**Comment by moidx** [comment]
I prefer this version as the other one has not been kept in sync with newer versions of Vivado. Others may have different opinions. We can probably take a quick poll on slack.

**Comment by mundaym** [comment]
I'm in favour of this.

**Comment by jrvanwhy** [comment]
@mundaym Is there anything preventing us from merging this now? I believe I've addressed all review comments.

**Comment by mundaym** [comment]
Sorry, just realised I had some pending review comments. There is nothing major there so feel free to merge and address in a follow up PR.

File: doc/getting_started/install_vivado/index.md [hunk 2]
```diff
@@ -2,101 +2,26 @@
...
-    Click "Next" to continue.
-
-    ![Vivado installation step 7](img/step7.png)
-
-11. Double-check the installation summary and click on "Install" to start the installation process.
+_**Vivado Version:** The recommendation is to use Vivado {{< tool_version "vivado" >}}._
 
-    ![Vivado installation step 8](img/step8.png)
+See [Download and
+Installation](https://docs.xilinx.com/r/2020.2-English/ug973-vivado-release-notes-install-license/Download-and-Installation)
```
[the remark this answers is not in the document]
Comment (jrvanwhy) [reply doc/getting_started/install_vivado/index.md]: Okay, I did that replacement. Is there a way for me to see how opentitan.org will render it?

File: doc/getting_started/install_vivado/index.md [hunk 3]
```diff
@@ -2,101 +2,26 @@
...
+When asked what edition to install, choose "Vivado HL Design Edition". On the
+feature selection screen, select at least the following features:
 
-    ![Vivado installation step 9](img/step9.png)
+![Vivado features selection screen](features.png)
 
-13. As soon as the installation has completed close the installer and you're now ready to use Vivado!
+After installing Vivado, you will need to add Vivado's paths to your shell
+environment. See [Launching the Vivado IDE from the Command Line on Windows or
+Linux](https://docs.xilinx.com/r/2020.2-English/ug892-vivado-design-flows-overview/Launching-the-Vivado-IDE-from-the-Command-Line-on-Windows-or-Linux)
```
[the remark this answers is not in the document]
Comment (jrvanwhy) [reply doc/getting_started/install_vivado/index.md]: Done.

File: doc/getting_started/install_vivado/index.md [hunk 4]
```diff
@@ -2,101 +2,26 @@
...
 ---
 
-## About Xilinx Vivado
-
-Generating a bitstream for Xilinx devices requires a Vivado installation.
-Vivado is provided by Xilinx, it is freeware for certain (smaller) FPGA devices but requires a commercial license for larger FPGAs.
-The free version is called "WebPACK", the commercial version "Design Edition".
-The installation instructions below are valid for both installation methods.
+Generating a bitstream for Xilinx devices requires a
+[Vivado](https://www.xilinx.com/products/design-tools/vivado.html) installation.
```
[the remark this answers is not in the document]
Comment (jrvanwhy) [reply doc/getting_started/install_vivado/index.md:6]: Done.

File: doc/getting_started/install_vivado/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jrvanwhy) [reply doc/getting_started/install_vivado/index.md]: My question is answered at https://docs.opentitan.org/doc/ug/documentation/

I checked and it renders correctly.

**Pull request merged**